src/email/logs.controller.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Email log Controller', function() {
+    var scope, controller;
+
+    beforeEach(module('motech-email'));
+    beforeEach(inject(function($rootScope, $controller){
+        scope = $rootScope.$new();
+        controller = $controller('EmailLogsController', {$scope: scope});
+    }));
+
+    it('should return good data from scope.checkboxes', function() {
```

Comment:
@DamianNowakSoldevelo Change the name of the test to: `'should set scope.checkboxes'`